dataland-frontend/src/components/pages/ViewFrameworkData.vue

```diff
@@ -64,7 +64,7 @@ export default defineComponent({
         DataTypeEnum.Sfdr,
         DataTypeEnum.P2p,
         DataTypeEnum.Sme,
-      ];
+      ] as string[];
```

Comment:
was this added just because of linting issues? or does this change the behavior in some way?

Reply:
Behaviour ändert sich nicht.
Das ist damit die IDE aufhört es rot zu unterstreichen, wenn man unten den `.push()` in diese Liste macht.